lib/modules/platform/gitlab/types.ts

```diff
@@ -37,6 +37,38 @@ export interface GitLabMergeRequest {
   };
 }
 
+export interface GitlabCommit {
+  id: string;
+  short_id: string;
+  title: string;
+  author_name: string;
+  author_email: string;
+  committer_name: string;
+  committer_email: string;
+  created_at: string;
+  message: string;
+  committed_date: string;
+  authored_date: string;
+  parent_ids: string[];
+  last_pipeline: LastPipeline;
```

Comment:
can this be null when no pipeline was run?

Reply:
Yes, i fix it in the latest commit